src/routes.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+const logger = require('./logger')
+const {expectedEndpointsDirectories} = require('./constants')
+
+exports.createRoutes = router => {
+  validateDirectoryStructure()
+  setUpRoutes(router)
+}
+
+const validateDirectoryStructure = () => {
+  if (!fs.existsSync(path.resolve(__dirname, '..', 'endpoints'))) {
+    logger.error('Directory "endpoints" not found in project root')
+    process.exit(1)
```

Comment:
Sure thing - I'll think about how to implement this - the reason I put the hard exit though is because this mediator really shouldn't be used if this bit fails. But it's definitely impossible to test now...

Reply:
@Zooloo2014 I agree with @MattyJ007 on this one, this validation is done as part of the "start up" process. If validation fails, this should be seen as a failed start up since the mediator will never be in a usable state until someone fixes the issue and restarts the service.